pkg/quarterdeck/db/migrations/0002_default_data.sql

```diff
@@ -9,29 +9,31 @@ INSERT INTO roles (id, name, description, created, modified) VALUES
 ;
 
 INSERT INTO permissions (id, name, description, allow_api_keys, allow_roles, created, modified) VALUES
-    (1, 'organization:edit', 'Can make changes to the details of an organization', false, true, datetime('now'), datetime('now')),
-    (2, 'collaborators:add', 'Can add users to an organization and change their roles', false, true, datetime('now'), datetime('now')),
-    (3, 'collaborators:remove', 'Can remove users from an organization', false, true, datetime('now'), datetime('now')),
-    (4, 'collaborators:read', 'Can see the collaborators for an organization', false, true, datetime('now'), datetime('now')),
-    (5, 'projects:edit', 'Can create and update projects in the organization', false, true, datetime('now'), datetime('now')),
-    (6, 'projects:delete', 'Can delete projects from the organization along with their data', false, true, datetime('now'), datetime('now')),
-    (7, 'projects:read', 'Can view projects in the organization', false, true, datetime('now'), datetime('now')),
-    (8, 'apikeys:edit', 'Can create and update api keys in a project', false, true, datetime('now'), datetime('now')),
-    (9, 'apikeys:delete', 'Can delete api keys from a project', false, true, datetime('now'), datetime('now')),
-    (10, 'apikeys:read', 'Can view api keys inside of a project', false, true, datetime('now'), datetime('now')),
-    (11, 'topics:create', 'Can create topics in a project', true, true, datetime('now'), datetime('now')),
-    (12, 'topics:edit', 'Can modify topic metadata and mark as read only', true, true, datetime('now'), datetime('now')),
-    (13, 'topics:destroy', 'Can destroy topics and all their data', true, true, datetime('now'), datetime('now')),
-    (14, 'topics:read', 'Can view the topics for a project', true, true, datetime('now'), datetime('now')),
-    (15, 'metrics:read', 'Can view the metrics of topics in the specified project', true, true, datetime('now'), datetime('now')),
-    (16, 'publisher', 'Is allowed to publish events to the topics in the project', true, false, datetime('now'), datetime('now')),
-    (17, 'subscriber', 'Is allowed to subscribe to topics in the project to consume events', true, false, datetime('now'), datetime('now'))
+    (1, 'organizations:create', 'Can create new organizations', false, true, datetime('now'), datetime('now')),
+    (2, 'organizations:delete', 'Can delete organizations and their data', false, true, datetime('now'), datetime('now')),
+    (3, 'organizations:list', 'Can view the list of organizations', false, true, datetime('now'), datetime('now')),
+    (4, 'organizations:edit', 'Can make changes to organizations', false, true, datetime('now'), datetime('now')),
+    (5, 'organizations:detail', 'Can view details of an organization', false, true, datetime('now'), datetime('now')),
```

Comment:
This is an area of uncertainty for me, should we create a new role which has the top level organization permissions?

Reply:
I think the "Owner" role is well suited to the organization privilege -- so I would just use that role for these.